### PR TITLE
Add Debian docker-testrunner Dockerfile

### PR DESCRIPTION
**Pull request by MichaelSimons**
Related to #dotnet/dotnet-docker-nightly#521

### CODE REVIEW

File: src/debian/stretch/docker-testrunner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        gnupg2 \
+        software-properties-common \
+    && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
+    && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/debian $(lsb_release -cs) stable" \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+        docker-ce=17.12.0~ce-0~debian \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Powershell
```
Comment (ravimeda) [inline src/debian/stretch/docker-testrunner/Dockerfile]: `Powershell` should be `PowerShell`

File: src/debian/stretch/docker-testrunner/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Dockerfile used to create a testrunner image that can perform Docker operations.
+#     docker run --rm -v /var/run/docker.sock:/var/run/docker.sock testrunner pwsh -File xyz.ps1
```
Comment (ravimeda) [inline src/debian/stretch/docker-testrunner/Dockerfile]: Minor - `#     docker` extra space in between can be removed

Comment (MichaelSimons) [reply src/debian/stretch/docker-testrunner/Dockerfile]: Replaced with `Usage:  `

File: src/debian/stretch/docker-testrunner/Dockerfile (hunk 1 above)
Comment (MichaelSimons) [reply src/debian/stretch/docker-testrunner/Dockerfile]: Fixed

**Pull request merged**